core/src/main/scala/org/apache/spark/network/netty/NettyBlockTransferService.scala

```diff
@@ -18,11 +18,14 @@
 package org.apache.spark.network.netty
 
 import java.nio.ByteBuffer
+import java.util
```

Comment:
if you are trying to avoid confusion w/ scala's hashmaps, I think our convention is to rename w/ "J" prefix

```
import java.util.{HashMap => JHashMap}
```

Reply:
OK, I will change it.